posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -512,6 +512,13 @@ def _counts_breakdown_value(self):
                 return ast.Field(chain=["properties", "$browser"])
             case WebStatsBreakdown.OS:
                 return ast.Field(chain=["properties", "$os"])
+            case WebStatsBreakdown.VIEWPORT:
+                return ast.Tuple(
+                    exprs=[
+                        ast.Field(chain=["properties", "$viewport_width"]),
+                        ast.Field(chain=["properties", "$viewport_height"]),
+                    ]
+                )
```

Comment:
@robbie-c Thoughts on whether we should viewport or window width/height? Our SDK captures both already

I believe viewport is more useful but it outputs some weird numbers because people are more used to screen sizes rather than viewport sizes.

I could go either way depending on what you think

Reply:
Viewport sounds good to me. It's what people will use for media queries and such.

Reply:
You could try this function instead of a tuple, idk if it's better but could be worth trying https://clickhouse.com/docs/en/sql-reference/functions/string-functions#concatwithseparatorassumeinjective

Reply:
Oh, I merged without checking this, you're probably correct. Let me test which one is faster :)

Reply:
Here's what the current query looks like
![image](https://github.com/user-attachments/assets/867a826f-95c6-481c-845a-9adfcafcbc6f)


Here's what `concatWithSeparator` would look like
![image](https://github.com/user-attachments/assets/ad3d7fe4-dbd5-437d-8ced-6d09be50cc2d)

Pretty similar, so in the interest of time, I'll keep this as is, but this is definitely something we should use in the future :)